packages/components/select/src/option-group.vue

```diff
@@ -57,25 +57,24 @@ export default defineComponent({
       children.value.some((option) => option.visible === true)
     )
 
+    const isOption = (node) =>
+      node.type?.name === 'ElOption' && node.component?.proxy
+
     // get all instances of options
     const flattedChildren = (node) => {
+      const Nodes = isArray(node) ? node : [node]
```

Comment:
There is an `ensureArray` function in `@element-plus/utils`, which is more convenient to use directly.
```suggestion
      const Nodes = ensureArray(node)
```